torch/ao/quantization/observer.py

```diff
@@ -146,7 +146,7 @@ class ObserverBase(ABC, nn.Module):
         or static quantization
     """
 
-    def __init__(self, dtype, is_dynamic=False):
+    def __init__(self, dtype, is_dynamic: bool = False):
```

Comment:
Curious—are you running another lint pass on these aside from removing unused type ignores?

Reply:
In this case, that implicit `Any` just caught my eye, and I figured I add some actual type information while I'm at it.